demos/redis-cluster/Implementation Guide

```diff
@@ -0,0 +1,115 @@
+Creating image and pushing to openshift registry.
```

Comment:
This should be in markdown format and renamed Implementation_Guide.adoc. Then should be linked from somewhere in the README.